app/controllers/poms_controller.rb

```diff
@@ -10,8 +10,16 @@ def index
     active_poms, inactive_poms = poms.partition { |pom|
       %w[active unavailable].include? pom.status
     }
-    @active_poms = active_poms.map { |p| PomPresenter.new(p) }
-    @inactive_poms = inactive_poms.map { |p| PomPresenter.new(p) }
+
+    if @prison.womens?
+      # poms are split into groups - 1) probation 2) prison & are sorted by last name
```

Comment:
We have been creating new controllers using routing rather than splitting behaviours like this.